packages/api/internal/auth/middleware.go

```diff
@@ -83,8 +83,6 @@ func (a *commonAuthenticator[T]) Authenticate(ctx context.Context, input *openap
 		return fmt.Errorf("%s %w", a.errorMessage, err)
 	}
 
-	telemetry.ReportEvent(ctx, "api key extracted")
```

Comment:
This was spamming me locally, and seemed useless. Happy to bring it back if it's useful though.